app/screens/home/search/results/file_card.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {View, Text, TouchableOpacity} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import FormattedDate from '@components/formatted_date';
+import FileIcon from '@components/post_list/post/body/files/file_icon';
+import {useTheme} from '@context/theme';
+import {getFormattedFileSize} from '@utils/file';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+type Props = {
+    fileInfo: FileInfo;
+}
+
+const hitSlop = {top: 24, bottom: 24, left: 24, right: 24};
```

Comment:
24 may be too much.. the best way to test this is in your device, press the icon and then move your finger to the top/left/right/bottom and see where does the opacity returns to normal, with that you can tweak this values. But I have not tried it myself, so it could be that this is fine.

I'll defer this to @matthewbirtch

Reply:
@matthewbirtch, I set this at `24` after playing around with it for a while.  This seems highly subjective, but I "assumed" a medium sized finger tip (mine) and set it to a value that I thought kept me from unintentially tapping the full card instead of the `...`.  I can also defer to your input if you'd like it changed.

Reply:
I think it might be too big - it looks like I can actually tap outside of the container on the space between two items and it taps the 3-dot icon. the hitslop probably doesn't need to extend beyond the bounds of the card